### DISCUSSION TITLE
[앱] 메모앱 추천합니다.(Obsidian)

### DISCUSSION THREAD

**Discussion by lina1919** [SWM-Space-Odyssey/WeddingMate_BackEnd#110] [Show and tell]
### 메모앱 추천
최근 발견한 제 최애 메모앱 옵시디언을 추천합니다.
상당히 개발자 친화적이라 저희 팀원들도 사용해봤으면 하는 마음에 discussion에 올립니다.
 사용법이나 추천 플러그인들은 제 블로그에 올렸으니 다들 한번 읽어보시고 사용해보세요~!

[https://yurimy.tistory.com/34](url)

**Comment by nitronium102** [comment]
저도 오늘부터 옵시디언 생활 시작해보겠습니다💪💪 잠깐 사용해봤는데도 로딩 속도가 빨라서 너무 좋네요~~
![image](https://github.com/SWM-Space-Odyssey/WeddingMate_BackEnd/assets/68282057/f0dbfbe8-ddf4-4d85-9b90-e249a121e608)
